clients/venice-thin-client/src/main/java/com/linkedin/venice/client/stats/BasicClientStats.java

```diff
@@ -39,6 +45,7 @@ public static BasicClientStats getClientStats(
 
   protected BasicClientStats(MetricsRepository metricsRepository, String storeName, RequestType requestType) {
     super(metricsRepository, storeName, requestType);
+    this.isMetaSystemStore = storeName.startsWith(SYSTEM_STORE_NAME_PREFIX);
     requestSensor = registerSensor("request", requestRate);
```

Comment:
We are still registering sensor for sys store here?

Reply:
this code has been reverted